pkgs/development/libraries/opencolorio/default.nix

```diff
@@ -11,7 +11,9 @@ stdenv.mkDerivation rec {
 
   outputs = [ "bin" "out" "dev" ];
 
-  buildInputs = [ cmake unzip ];
+  buildInputs = [ cmake unzip ] ++ lib.optional stdenv.isDarwin boost;
```

Comment:
This is a bit strange, why does opencolorio depend on Boost on macOS but not on other platforms?

Reply:
Yeah boost is an optional dependency of opencolorio. It will be used by opencolorio automatically if it's found. MacOS builds seem to need it for some compatibility libraries while Linux seems to not need it.